### PR TITLE
feat(cli): add scanner flags that ignore vulnerabilities with no fix and ignore vulnerabilities by CVE ID

### PR DESCRIPTION
**Pull request by pbalogh-sa**
Signed-off-by: Peter Balogh <p.balogh.sa@gmail.com>
fixes: #70 
Two new flags for scanner command:
- ignore vulnerabilities that don't have any fix
- ignore vulnerabilities that are in the command line list.

### CODE REVIEW

File: cli/cmd/scan.go [hunk 1]
```diff
@@ -78,6 +79,8 @@ func init() {
 		"export vulnerability scan results to the backend")
 	scanCmd.Flags().Bool("cis-docker-benchmark-scan", false,
 		"enables CIS docker benchmark scan. (relevant only for image source type)")
+	scanCmd.Flags().Bool("ignore-no-fix", false, "ignore vulnerabilities that don't have any fix")
```
Comment (FrimIdan) [inline cli/cmd/scan.go:82]: `ignore vulnerabilities that are not fixed` WDYT?

File: cli/pkg/scanner/common/common.go [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import "github.com/openclarity/kubeclarity/shared/pkg/scanner"
+
+// TODO can be multiple fix version?
+func GetFixVersion(vulnerability scanner.Vulnerability) string {
```
Comment (FrimIdan) [inline cli/pkg/scanner/common/common.go:21]: maybe should be put `GetFixVersion` in `/github.com/openclarity/kubeclarity/shared/pkg/scanner/vulnerability.go`?

Comment (pbalogh-sa) [reply cli/pkg/scanner/common/common.go]: I agree with you. thx

File: cli/cmd/scan.go [hunk 3]
```diff
@@ -78,6 +79,8 @@ func init() {
 		"export vulnerability scan results to the backend")
 	scanCmd.Flags().Bool("cis-docker-benchmark-scan", false,
 		"enables CIS docker benchmark scan. (relevant only for image source type)")
+	scanCmd.Flags().Bool("ignore-no-fix", false, "ignore vulnerabilities that are not fixed")
```
Comment (akpsgit) [inline cli/cmd/scan.go:82]: maybe: ignore vulnerabilities that have no fix available

Comment (pbalogh-sa) [reply cli/cmd/scan.go]: ack

**Review by akpsgit — APPROVED** [review]

**Pull request merged**